framework/db/conditions/ConditionInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
Standard Yii header missing.

Reply:
It will be added during release process automatically